serverless.yml

```diff
@@ -30,6 +36,8 @@ functions:
           method: post
     environment:
       ALMA_SECRET_KEY_NAME: /library-api-gateway/webhook-handler/shared-secret
+      AWS_REGION: ${opt:region}
```

Comment:
This shouldn't need to be specified as `AWS_REGION` is added as an environment variable by AWS when the lambda starts.

Reply:
This is fixed in https://github.com/lulibrary/alma-webhook-handler/pull/9/commits/a16cad8ddae31ae9d7f4f6e996b2e76b3ebf6657